### PR TITLE
RHDEVDOCS-5352: Documenting 1.10.3 RN

### PR DESCRIPTION
**Pull request by shipsing**
**Purpose**: To resolve the following issue: 
* https://issues.redhat.com/browse/RHDEVDOCS-5352

**Aligned team**: DevTools

**OCP version this PR applies to (cherrypicking)**: enterprise 4.11 and later versions

**Content for preview**:  https://60535--docspreview.netlify.app/openshift-enterprise/latest/cicd/pipelines/op-release-notes.html#release-notes-1-10-3_op-release-notes

**SME review** :  @khrm 

**QE review**:  @VeereshAradhya @ppitonak 

**Peer review**: @max-cx

### CODE REVIEW

**Review by khrm — COMMENTED** [review]
/lgtm

**Review by max-cx — APPROVED** [review]
LGTM on the diffs and the preview.

**Comment by shipsing** [comment]
@VeereshAradhya @ppitonak Please review the PR.

File: modules/op-release-notes-1-10.adoc [hunk 1]
```diff
@@ -176,3 +176,19 @@ With this update, {pipelines-title} General Availability (GA) 1.10.2 is availabl
...
 
 Before this update, an issue in the Tekton Operator prevented the user from setting the value of the `enable-api-fields` flag to `beta`. This update fixes the issue. Now, you can set the value of the `enable-api-fields` flag to `beta` in the `TektonConfig` CR.
+
+[id="release-notes-1-10-3_{context}"]
+== Release notes for {pipelines-title} General Availability 1.10.3
+
+With this update, {pipelines-title} General Availability (GA) 1.10.3 is available on {product-title} 4.11, 4.12, and 4.13.
+
+[id="new-features-1-10-3_{context}"]
+=== New features
```
Comment (concaf) [inline modules/op-release-notes-1-10.adoc:186]: this is a bugfix

File: modules/op-release-notes-1-10.adoc [hunk 2]
```diff
@@ -176,3 +176,19 @@ With this update, {pipelines-title} General Availability (GA) 1.10.2 is availabl
...
+
+With this update, as a cluster administrator, you can customize the following performance configuration fields in the `TektonConfig` CR based on your needs:
```
Comment (concaf) [inline modules/op-release-notes-1-10.adoc:188]: before this update, operator did not expose the following performance configuration fields, this is now fixed
^^^ something like this

**Review by VeereshAradhya — APPROVED** [review]

**Review by concaf — APPROVED** [review]

**Review by concaf — APPROVED** [review]

**Pull request merged**